.github/workflows/sdk-generator.yml

```diff
@@ -0,0 +1,37 @@
+name: Generate Client SDKs from OpenAPI
+
+on:
+    push:
+        branches:
+            - main
```

Comment:
This should be master. And adding 'workflow_dispatch:' would be nice since it allows for manual execution.

Reply:
done 👍🏼